packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -66,12 +66,12 @@ export abstract class TelemetryLogger implements ITelemetryLogger {
             event.stack = errorAsObject.stack;
             event.error = errorAsObject.message;
 
-            // Error message can container PII information.
+            // Error message can contain PII information.
             // If we know for sure it does, we have to not log it.
             if (error.containsPII) {
                 event.error = "Error message was removed as it contained PII";
-            } else if (error.getCustomProperties) {
-                const customProps: ITelemetryProperties = error.getCustomProperties();
+            } else if (error.getProperties) {
```

Comment:
getProperties is too generic name that can have collision with some other object that has nothing to do with LoggingError, and thus we will record potential PII data.
We either need more unique name here, or move away from such naming at all, and rather do (error instanceof LoggingError) test. With later, we do not care about having this method, we could just do { ...error } (though that has an impact of enumerating all keys, whereas having method theoretically allows us to filter).

Reply:
Renamed it to getTelemetryProperties.